ansible/validate.yml

```diff
@@ -27,4 +27,26 @@
     - import_role:
         name: opendistro
         tasks_from: validate.yml
-      tags: validate
\ No newline at end of file
+      tags: validate
+
+- name: Validate rebuild configuration
+  hosts: openhpc
+  gather_facts: false
+  tags: rebuild
+  tasks:
+    - block:
+      - stat:
+          path: "{{ openhpc_rebuild_clouds }}"
+        register: openhpc_clouds_path
+      - name: Check OpenRC file (openhpc_rebuild_clouds) exists
+        assert:
+          that: openhpc_clouds_path.stat.exists
+          fail_msg: "openrc file {{ openhpc_rebuild_clouds }} on localhost (specified by `openhpc_rebuild_clouds`) does not exist"
+      - name: Check OpenRC file (openhpc_rebuild_clouds) is usable
```

Comment:
Add no_log here so that we don't leak a token in stdout.

Reply:
All I can find on the [manpage](https://docs.openstack.org/python-openstackclient/latest/cli/man/openstack.html#manpage) is:
> --log-file <LOGFILE>¶
>  Specify a file to log output. Disabled by default.

Reply:
Validate tasks now defined in collection's role which does set this.